scripts/plot_ma_key_variables.py

```diff
@@ -47,12 +47,44 @@
 
 
 version: list = ["v2", "v3", "merged"]
-plot_type: list = ["Inflow_centrality", "Outflow_centrality"]
+plot_type_eigen: list = ["Inflow_centrality", "Outflow_centrality"]
+plot_type_eigen_undirected: list = ["eigenvector_centrality"]
+plot_type_cluster: list = ["clustering_coefficient"]
 
 # generate the graph dict
 graph_dict = {}
+
+for v in version:
```

Comment:
why multiple for v in version? why don't we put them all together? 

some inline comments what each chunk do would help @lyc0603